Testscripts/Linux/utils.sh

```diff
@@ -2258,9 +2258,12 @@ function add_sles_benchmark_repo () {
 	source /etc/os-release
 	IFS='- ' read -r -a array <<< "$VERSION"
 	repo_url="https://download.opensuse.org/repositories/benchmark/SLE_${array[0]}_${array[1]}/benchmark.repo"
-	LogMsg "add_sles_benchmark_repo - $repo_url"
-	zypper addrepo $repo_url
-	zypper --no-gpg-checks refresh
+	wget $repo_url -O /dev/null -o /dev/null
+	if [ $? -eq 0 ]; then
+		LogMsg "add_sles_benchmark_repo - $repo_url"
+		zypper addrepo $repo_url
+		zypper --no-gpg-checks refresh
+	fi
 	return 0
```

Comment:
If it repo_url does not exists, do we need to still return 0 ? Is caller not handling return status ?